src/app/components/Article/Metadata/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import { shape, string } from 'prop-types';
+
+const Metadata = ({ canonicalLink, htmlAttributes, title }) => (
+  <Helmet htmlAttributes={htmlAttributes}>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, minimum-scale=1"
+    />
+    <title>{title}</title>
+    <link rel="canonical" href={canonicalLink} />
+  </Helmet>
+);
+
+Metadata.propTypes = {
+  canonicalLink: string.isRequired,
+  htmlAttributes: shape({}).isRequired,
```

Comment:
This `htmlAttributes` prop-type should be defined with a `shape` that includes `amp` and `lang`.